pom.xml

```diff
@@ -535,6 +548,11 @@
         </exclusions>
       </dependency>
       <dependency>
+        <groupId>org.hammerlab</groupId>
+        <artifactId>spark-bam_2.10</artifactId>
```

Comment:
`spark-bam_2.10`, you say 🤔 😁

is this here in anticipation of a world where that exists? j/w / trying to follow